File: burakcanyildiz/lib/Models/cevap.dart
import 'package:burakcanyildiz/Models/soru.dart';

class Cevap {
  int id;
  int soruId;
  Soru soru;
  String name;
  int puan;

  Cevap({this.id, this.soruId,this.soru,this.name,this.puan});

  Cevap.fromJson(Map<String, dynamic> json) {
    id = json['id'];
    soruId = json['soruId'];
    soru = json['soru'] != null ? Soru.fromJson(json['soru']) : null;
    name = json['name'];
    puan = json['puan'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['id'] = this.id;
    data['soruId'] = this.soruId;
    data['soru'] = this.soru;
    data['name'] = this.name;
    data['puan'] = this.puan;
    return data;
  }
}

File: burakcanyildizbackend/Models/Cevap.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace burakcanyildizbackend.Models
{
    public class Cevap : BaseEntity
    {

        [Key]

        public int Id { get; set; }

        public int SoruId { get; set; }

        public Soru Soru { get; set; }

        public string Name { get; set; }

        public int Puan { get; set; }

    }
}
